lib/school_web/views/protocol_view.ex

```diff
@@ -0,0 +1,14 @@
+defmodule SchoolWeb.ProtocolView do
+  use SchoolWeb, :view
+
+  def render("show.json", %{data: data}) do
+    %{person: person, number_of_courses: number_of_courses} = data
```

Comment:
You can pattern match on the function
render("show.json", %{data: %{person: person, number_of_courses: number_of_courses}})